views/partials/head.ejs

```diff
@@ -1,7 +1,6 @@
 <!--views/partials/head.ejs-->
 <meta charset='UTF-8'>
-<title>Welcome to learndb.dev </title>
-
+<title></title>
```

Comment:
couldy ou delete this tag if you don't need it?

Reply:
okay